payment/controllers.go

```diff
@@ -30,6 +31,131 @@ func Router(service *Service) chi.Router {
 	return r
 }
 
+// KeyRouter handles management of keys
+func KeyRouter(service *Service) chi.Router {
+	r := chi.NewRouter()
+	if os.Getenv("ENV") != "local" {
+		r.Method("GET", "/{merchantId}", middleware.SimpleTokenAuthorizedOnly(GetKeys(service)))
```

Comment:
would you mind separating this so that we just have a handler, then overwrite it if a condition is met?

Reply:
https://github.com/brave-intl/bat-go/pull/285/commits/2b658101144a0dba17c8d0194fbbc8dbd3021cd7